app/frontend/stylesheets/user_dashboard/_show.scss

```diff
@@ -0,0 +1,153 @@
+#user-dashboard {
+  height: 100vh;
+  margin: 0;
+
+  .log-out {
+    margin-left: 20px;
+    text-align: right;
+  }
+
+  .navbar {
+    margin-top: 25px;
+    height: 100px;
+  }
+
+  .logo {
+    padding-top: 15px;
+    filter: invert(100%);
+  }
+
+  .background {
+    opacity: 0.9;
+    position: absolute;
+    right: 50%;
+    top: 30%;
+    translate: 50% -25%;
+    width: 170%;
+    height: 100%;
+    z-index: -20;
+  }
+
+  .user-dashboard-container {
+    display: flex;
+    // align-items: center;
+    justify-content: center;
+    margin: auto;
+    width: 1300px;
+
+  }
+
+  .user-profile-container {
+    background-color: #C6D5CF ;
+    height: 500px;
+    width: 350px;
+    margin-top: 100px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-start;
+    border-radius: 35px;
+    box-shadow: 20px 20px 10px rgba(0, 0, 0, 0.2);
+    position: relative;
+  }
+
+  .leaf {
+    position: absolute;
+    z-index: 10;
+
+    &#top {
+      height: 400px;
+      left: -40px;
+      top: -90px;
+      transform: rotateY(180deg) rotate(-15deg);
+      width: 250px;
+    }
+
+    // &#bottom {
+    //   height: 350px;
+    //   left: 180px;
+    //   bottom: -90px;
+    //   transform: rotate(60deg);
+    //   width: 200px;
+    // }
+  }
+
+
+  .profile-icon {
+    width: 120px;
+    padding: 40px 0 50px 0;
+  }
+
+  .user-profile-info {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding-top: 90px;
+
+
+    h2 {
+      font-size: 30px;
+      width: 50%;
+      text-align: center;
+    }
+
+    p {
+      font-size: 20px;
+    }
+
+  }
+
+  .user-options-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 30px;;
+  }
+
+  .user-options-boxes {
+    display: flex;
+    align-items: end;
+    justify-content: end;
+    flex-flow: row wrap;
+    width: 900px;
+    margin-left: 15px;
+
+    .box-top {
+      margin-top: 70px;
+      margin-bottom: 90px;
+    }
+  }
+
+  .user-options-info {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: #99ABB8;
+    border-radius: 30px;
+    box-shadow: 20px 20px 10px rgba(0, 0, 0, 0.2);
+    width: 370px;
+    margin: 0 auto;
+    height: 205px;
+  }
+
+  .user-options-info {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    font-size: 20px;
+
+    .options-icon {
+      margin-bottom: 15px;
+    }
+  }
+
+
+  .fa-regular {
+    font-size: 70px;
+    padding-bottom: 20px;
+  }
```

Comment:
never edit font awesome's classes directly.
Make a new one instead. Or use the `i` selector